app/backend/src/clj/teet/user/user_db.clj

```diff
@@ -24,6 +24,14 @@
   [db user]
   (:db/id (du/entity db (user-model/user-ref user))))
 
+(defn user-has-logged-in?
```

Comment:
@aleksievelizhanin this function is for you

Reply:
Thx!